tests/containers/cockroach.py

```diff
@@ -15,33 +19,126 @@
 log = logging.getLogger(__name__)
 
 
+COCKROACH_IMAGE = 'cockroachdb/cockroach:v2.0.7'
+
+
 class ContainerCockroach(ContainerBase):
 
-    imagename = 'cockroachdb/cockroach:v1.1.9'
-    container_name = 'bouncer-test-cockroach'
     # Listen on all interfaces on the host machine so that CockroachDB is
     # reachable through the Docker bridge network.
     listen_ip = '0.0.0.0'
 
-    def __init__(self):
+    # Port on which the cockroachdb listens inside the container
+    db_port = 26257
+
+    sql_alchemy_base_url = 'cockroachdb://root@bouncer-test-hostmachine'
+
+    def __init__(
+            self,
+            container_name='bouncer-test-cockroach',
+            imagename=COCKROACH_IMAGE,
+            port=26257,
+            volume_binds={},
+            ):
+        """
+        Args:
+            imagename (str): Name of the image that will be used to launch
+                the cockroachdb container.
+            port (int): Number on which will be cockroachdb available to
+                other containers.
+            volume_binds (dict): Bind volumes to a docker container running
+                cockroachdb.
+                See: https://docker-py.readthedocs.io/en/1.10.0/volumes/
```

Comment:
Since we're updating the `docker` package, can we update this to equivalent docs?